### DISCUSSION TITLE
Block write permissions for non-administrator users

### DISCUSSION THREAD

**Discussion by Dantemau** [immich-app/immich#12836] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be great if a feature could be implemented where the administrator could lock and unlock permissions to upload and delete files for certain users. 

For example: I use Immich with more than 5 people, and I created a public profile (everyone knows the username and password) but I wouldn't like anyone who logs in with that public profile to delete existing images/videos or upload new files.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#8236
